README.md

```diff
@@ -1 +1,70 @@
-# ToDoList
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do-list
+
+In this project, I built a simple HTML list of To Do tasks. The list is styled according to this [template](https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/). This simple web page was built using webpack and served by a webpack dev server.
+
+![screenshot](screenshot.png)
+
```

Comment:
- [OPTIONAL] The link to the screenshot is broken, please kindly fix that as this will make your `readme file` look more professional and will portray you as someone who pays attention to details.